ansible/roles/rke/tasks/main.yml

```diff
@@ -34,8 +34,8 @@
     remote_src: yes
     extra_opts: [--strip-components=1]
   with_items:
-    - https://storage.googleapis.com/kubernetes-helm/helm-v2.14.3-linux-amd64.tar.gz
-    - https://github.com/rancher/cli/releases/download/v2.2.0/rancher-linux-amd64-v2.2.0.tar.gz
+    - https://storage.googleapis.com/kubernetes-helm/helm-{{ helm_version }} -linux-amd64.tar.gz
```

Comment:
@ddl-retornam You've got some extra white-spaces in these two lines

Reply:
https://github.com/dominodatalab/ranchhand/pull/36